src/javatests/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructorTest.java

```diff
@@ -229,20 +260,26 @@ private TezTask createTezTaskWithCounters(String id, CountersHolder counters) {
   private Counters createMapReduceCounters(CountersHolder countersHolder) {
     Counters counters = new Counters();
 
-    countersHolder.groups().forEach(group -> {
-      Group countersGroup = counters.getGroup(group.name());
-      group.counters().forEach((key, value) -> {
-        Counter counterForName = countersGroup.getCounterForName(key);
-        counterForName.setValue(value);
-      });
-    });
+    countersHolder
+        .groups()
+        .forEach(
+            group -> {
+              Group countersGroup = counters.getGroup(group.name());
+              group
+                  .counters()
+                  .forEach(
+                      (key, value) -> {
+                        Counter counterForName = countersGroup.getCounterForName(key);
+                        counterForName.setValue(value);
```

Comment:
`counterForName` variable is redundant, I would suggest to inline it.

Reply:
Done